### ISSUE TITLE
Focus on node rather than way center

### ISSUE THREAD

**Issue by mrondr** [maproulette/maproulette3#2243]
**Describe the bug**
In some challenges (e.g., [Global - Sharp Turns](https://maproulette.org/browse/challenges/43006)) the user has to edit a node (or the neighbourhood of a node) within a way. 
MapRoulette 's classic mode focuses on the center of the way rather than on the node being the issue (in the example above, the vertex of a sharp angle). For particularly long ways this forces the user to zoom out to find the point being the issue and than zoom in on such node, needlessly.

**Expected behavior**
Focus on the node

**To Reproduce**
1. Go to a challenge such as https://maproulette.org/browse/challenges/43006
2. Find a way to be edited being long enough (e.g., the hairpin in a long mountain road)
3. See error

**Screenshots**
Current zoom
![Screenshot_20240116_082707](https://github.com/maproulette/maproulette3/assets/20371230/9d2319e3-64ff-4f87-9b97-6dab2b0058d8)
When zooming out
![Screenshot_20240116_082722](https://github.com/maproulette/maproulette3/assets/20371230/8ef8b64f-e0cd-4536-9141-969bb023f23f)
More appropriate zoom
![Screenshot_20240116_082733](https://github.com/maproulette/maproulette3/assets/20371230/b28b7dc6-78ac-41c8-849a-1a612c2f1969)

**Desktop (please complete the following information):**
 - OS: Manjaro
 - Browser: Firefox, but also Chromium
 - Version: Firefox 121

**Comment by CollinBeczak** [comment]
From what i could tell looking into this issue, how the initial view and the recentered view functions **_(or how it is supposed to function currently. There is an issue with the initial zoom that will need to be fixed so its not zoomed in too much like it is currently.)_** is that the map bounds is set to display all the coordinates provided for the task. Because you have a way(line) with many coordinates, and a node(marker) with one coordinate, it does look like the view is centered on the way, but thats not true.

Here is a task with multiple nodes, and a way. This is the recenter view.

<img width="809" alt="Screenshot 2024-01-28 at 3 32 35 PM" src="https://github.com/maproulette/maproulette3/assets/88843144/6488b592-3e4d-478c-ac35-8bf646464e81">

This is the zoomed in view to the those bottom points to confirm there is a way present.
<img width="833" alt="Screenshot 2024-01-28 at 3 34 31 PM" src="https://github.com/maproulette/maproulette3/assets/88843144/e6c5d7ba-e6ae-4ff9-b0ae-e7c84a6e29c6">

Basically the solution for your requested initial and recentered view of a task, like the one you provided, would be to exclude ways from the calculation of the view whenever both a node and ways are present in a task. Im not sure if that is something we would like to do yet. But we can fix the problem with the initial zoom issue first, and then once thats done, the developers and community can discuss if the solution i described is something we'd like to move forward with or if there is something else we can do.

**Comment by mrondr** [comment]
First of all, thank you using your time to look into this. I think I see your point and I guess the more convenient behavior wrt the central point of the map would depend on the task at hand. 
Perhaps if more people feel this way it could be a further customization element of the interface.
This is really not a bug, just a minor inconvenience and I certainly not asking you to do this based on my convenience (actually I wish I had the skills to do this myself). Again thank you a lot for the explanation.
